telegram/message.py

```diff
@@ -276,8 +331,6 @@ def to_dict(self):
         return data
 
     def _quote(self, kwargs):
-        """Modify kwargs for replying with or without quoting"""
```

Comment:
Please just keep these in place, they are so valuable when we read through the code in the future.